### DISCUSSION TITLE
posts/typst/typst_support

### DISCUSSION THREAD

**Discussion by giscus** [Wallbreaker5th/Wallbreaker5th.github.io#1] [Announcements]
# posts/typst/typst_support

https://wallbreaker5th.top/posts/typst/typst_support.html

**Comment by R-fzx** [comment]
能不能出个教程讲下怎么搭支持 Typst 的博客啊/kel

**Reply by Wallbreaker5th** [reply]
要用 [typst.ts](https://github.com/Myriad-Dreamin/typst.ts/)。

我的实现比较丑陋，是在前端加载并运行 Typst 的 WASM 版本，具体实现可以看一下我那两天的 commit；很大的问题是用户需要加载一整个 Typst 编译器和若干字体。

比较正经的实现可以参考 [hexo-renderer-typst](https://github.com/Myriad-Dreamin/typst.ts/tree/main/projects/hexo-renderer-typst)。

**Reply by R-fzx** [reply]
谢谢/bx
